test/functional/sqlsrv/sqlsrv_connStr.phpt

```diff
@@ -77,89 +95,10 @@ $c = connect(array(
 if ($c !== false) {
     fatalError("sqlsrv_connect(4) should have failed");
 }
-print_r(sqlsrv_errors());
+checkErrors($utf16Error);
 
 echo "Test succeeded.\n";
 
 ?>
 --EXPECTF--
```

Comment:
Glad you finally fixed this test, but one minor point. Please change from EXPECTF to EXPECT